activesupport/test/inflector_test_cases.rb

```diff
@@ -289,6 +289,7 @@ module InflectorTestCases
     "Fred’s"                => "Fred’s",
     "Fred`s"                => "Fred`s",
     "this was 'fake news'"  => "This Was 'Fake News'",
+    "new name(s)"           => "New Name(s)",
```

Comment:
Can you add another test case to show that entire words wrapped in parentheses are still capitalised:

```
    "new (names)"           => "New (Names)",
```

Reply:
Another edge-case occasionally seen in prose:

```
    "their (mis)deeds"           => "Their (Mis)deeds",
```

Reply:
I updated the code to also ignore the ")" to cover the edge-case mentioned by @jonathanhefner. Tests cases were updated as well.